content/guides/translation-connection/step2-understand-akeneo-pim.md

```diff
@@ -0,0 +1,66 @@
+# Understand Akeneo PIM
+
+## Versions and editions
+
+First, it is crucial for you to understand that Akeneo PIM exists in different flavours, depending on its version and edition.
+
+To know more about the different PIM versions, you can consult [this article](https://www.akeneo.com/blog/akeneo-introduces-a-simpler-product-release-cycle/) to understand our PIM release cycle.
+
+You can also consult our [release notes](https://www.akeneo.com/release-notes/) and our ["new key features released" page](https://help.akeneo.com/pim/serenity/whats-new.html) to understand what are our last Akeneo supported **versions** and the features we have added in each one.
+
+Then, in order to understand our different **editions** and their capabilities, you can consult our [dedicated page](https://www.akeneo.com/compare-editions/) on our website.
+
+## PIM features and data model
+
+>Let's play with Akeneo PIM!
+
+It is crucial to understand the PIM features and data model in order to create a great eCommerce connector.
+
+The best way to become an expert in Akeneo PIM is to play with it!  
+You can access our [Akeneo PIM Community Edition demo platform](https://demo.akeneo.com) (login/password: admin / admin).
+
+:::warning
+Please note that this Akeneo PIM CE demo instance is for exploratory purposes only and will only allow you to start getting familiar with Akeneo PIM. This instance is reset frequently and is therefore not intended to be used to build a connector.
+:::
+
+If you need an Akeneo PIM **Enterprise Edition** instance to start developing your connector, please [contact us](https://www.akeneo.com/contact/).
+
+### Acquire PIM knowledge
+
```

Comment:
To become an Akeneo PIM master, we have many different tools to help you.

Reply:
Good idea